client/stylesheets/_toCat.scss

```diff
@@ -5747,6 +5747,34 @@ textarea {
   font-size: 0.9rem !important;
 }
 
+.helpDiv h2, .helpDiv h3, .aboutDiv h2, .aboutDiv b, .faqdiv h2, .faqdiv h3, .safetydiv h2, .safetydiv h3, .reportdiv h2, 
+.reportdiv h3, .prohibiteddiv h2, .termsDiv h2, .contactdiv h2 {
+    font-weight: 400 !important;
+}
+
+.helpDiv, .aboutDiv, .faqdiv, .safetydiv, .reportdiv, .prohibiteddiv, .termsDiv, .contactdiv {
+    margin-bottom: 100px !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces